packages/phishing-controller/src/PhishingController.ts

```diff
@@ -195,56 +169,142 @@ export const phishingListKeyNameMap = {
   [ListKeys.PhishfortHotlist]: ListNames.Phishfort,
 };
 
+const controllerName = 'PhishingController';
+
+const metadata = {
+  phishingLists: { persist: false, anonymous: false },
+  whitelist: { persist: false, anonymous: false },
+  hotlistLastFetched: { persist: false, anonymous: false },
+  stalelistLastFetched: { persist: false, anonymous: false },
```

Comment:
These values reflect how we use this state today:

```suggestion
  phishingLists: { persist: true, anonymous: false },
  whitelist: { persist: true, anonymous: false },
  hotlistLastFetched: { persist: true, anonymous: false },
  stalelistLastFetched: { persist: true, anonymous: false },
```

We can see that the data is persisted here: https://github.com/MetaMask/metamask-extension/blob/55475df0b3b616f27f4ef4df9b0156acdf370f0e/app/scripts/metamask-controller.js#L1740

And we can see that none of the state properties are included in diagnostics here (which is what `anonymous` is planned to be used for): https://github.com/MetaMask/metamask-extension/blob/55475df0b3b616f27f4ef4df9b0156acdf370f0e/app/scripts/lib/setupSentry.js#L166